modules/core/src/core/animation-loop.js

```diff
@@ -242,6 +303,57 @@ export default class AnimationLoop {
     return this.props.onFinalize(...args);
   }
 
+  async enableWebVR() {
+    if (!('getVRDisplays' in navigator)) {
+      return false;
+    }
+
+    const displays = await navigator.getVRDisplays();
+    if (displays && displays.length) {
+      log.info(2, 'Found VR Displays', displays)();
+      // TODO: Consider resizing canvas to match vrDisplay.getEyeParameters()
+
+      this.vrDisplay = displays[0];
+      this.vrPresenting = false;
+      this.vrButton = createEnterVRButton({
```

Comment:
Nit; I still think it would be nice to be able to trigger split rendering without VR controller, to make it easy to test/debug this mode when you don't have devices handy.